lib/learn_test/strategies/rspec.rb

```diff
@@ -0,0 +1,89 @@
+module LearnTest
+  module Strategies
+    class Rspec < LearnTest::Strategy
+      def service_endpoint
+        '/e/flatiron_rspec/build/ironboard'
+      end
+
+      def detect
+        runner.files.include?('spec') && (spec_files.include?('spec_helper.rb') || spec_files.include?('rails_helper.rb'))
+      end
+
+      def configure
+        if format_option_present?
+          if dot_rspec.any? {|dot_opt| dot_opt.match(/--format|-f/)}
+            argv << dot_rspec.reject {|dot_opt| dot_opt.match(/--format|-f/)}
```

Comment:
i think ideally i'd like to remove `argv` from the strategies, that should really only be in the bin file and passed in through the options hash, so i wanna rm in the next pass